tests/unit/test_addtoken.py

```diff
@@ -0,0 +1,20 @@
+from unittest import mock
+from add_token import print_short
+
+def test_single(capsys):
+    with mock.patch('add_token.get_network_info', return_value=['10.23.53.54', ['10.23.53.54'], '32']):
+        print_short('t', 'c')
+        captured = capsys.readouterr()
+        output = captured.out.strip()
+        assert output == f"microk8s join 10.23.53.54:32/t/c"
+
+    
+def test_multiple(capsys):
+    with mock.patch('add_token.get_network_info', return_value=['d_ip', ['ip1', 'ip2', 'd_ip'], '4']):
+        print_short('t', 'c')
+        captured = capsys.readouterr()
+        all_outputs = captured.out.strip().split('\n')
+        all_outputs[0] == f"microk8s join d_ip:4/t/c"
+        assert all_outputs[1] == f"microk8s join ip1:4/t/c"
+        assert all_outputs[2] == f"microk8s join ip2:4/t/c"
+        assert len(all_outputs) == 3
```

Comment:
this can be a single `assert all_outputs == [cmd1, cmd2, cmd3]`

Reply:
Okay